R/axis.R

```diff
@@ -89,6 +106,16 @@ axis.gsplot <- function(object, ..., n.minor=0, tcl.minor=0.15, reverse=NULL) {
 }
 
 draw_axis <- function(object, side.name){
+  # method isn't made for multiple axis calls
+  which.axis <- which(names(object[[side.name]]) == 'axis')
+  if (length(which.axis) > 1){
+    for (axis.i in which.axis){
+      tmp <- object
```

Comment:
the ` object[[as.side_name(side)]][['axis']] <- append_replace(object[[as.side_name(side)]][['axis']], user_args[[fun.name]])` append_replace makes sure the new arguments override the existing ones.